### DISCUSSION TITLE
TinycoreLinux的运行问题

### DISCUSSION THREAD

**Discussion by cnyaojin** [v2fly/v2ray-core#506] [Q&A]
在使用vbox虚拟机，tinycorelinux运行V2ray时产生一个问题：
1. 证书不受信，x509: certificate signed by unknown authority。
    访问百度，淘宝和京东等直连的服务都没问题，使用vmess（outbound/balancer模式）都不行。将AllowInsecure设置为True后OK了。

奇葩的是：
在使用hyper-v虚拟机，tinycorelinux运行V2ray时产生两个问题：
1. 证书不受信，同上；
2. failed to process outbound traffic，原因未知。

上述情况有哪位大神有什么见解。
===================
2020/12/12 15:04:06 [Info] [3387931252] v2ray.com/core/app/proxyman/outbound: failed to process outbound traffic > v2ray.com/core/proxy/freedom: connection ends > context canceled
2020/12/12 15:04:06 [Info] [3387931252] v2ray.com/core/app/proxyman/inbound: connection ends > v2ray.com/core/proxy/http: connection ends > io: read/write on closed pipe
2020/12/12 15:04:09 [Info] [3264258753] v2ray.com/core/transport/internet/websocket: creating connection to tcp:xx.xx.xx:443
2020/12/12 15:04:09 [Info] [2349684323] v2ray.com/core/transport/internet/websocket: creating connection to tcp:xx.xx.xx:443
2020/12/12 15:04:10 [Info] [2349684323] v2ray.com/core/proxy/vmess/outbound: tunneling request to tcp:clients4.google.com:443 via tcp:xx.xx.xx:443
2020/12/12 15:04:11 [Info] [3264258753] v2ray.com/core/proxy/vmess/outbound: tunneling request to tcp:mtalk.google.com:443 via tcp:xx.xx.xx:443
2020/12/12 15:04:14 [Info] [4000357773] v2ray.com/core/app/proxyman/inbound: connection ends > v2ray.com/core/proxy/http: failed to read http request > read tcp 172.17.16.181:10087->172.17.74.135:65203: i/o timeout

====================

**Comment by kslr** [comment]
所以你的证书呢？

**Reply by cnyaojin** [reply]
在tinycore上安装证书不知道为什么总是有问题，使用centos和debian都可以直接信任证书的。在window10上运行也没问题。

**Comment by cnyaojin** [comment]
failed to process outbound traffic的问题解决了，问题还是时间。

虚拟机上的tinycore上显示的时间虽然 和window上显示的一样，但其实他们的时区不同，所以造成差异。

证书的问题还是木有解决。

待定。

**Reply by testcaoy7** [reply]
把ca-certificates.tcz这个包装上再看看
